src/api/controllers/misc.ts

```diff
@@ -133,10 +133,10 @@ export async function handleInternalVoiceStateUpdate(data: DiscordPayload) {
 }
 
 /** This function is the internal handler for the webhooks update event. Users can override this with controllers if desired. */
-export function handleInternalWebhooksUpdate(data: DiscordPayload) {
+export function handleInternalWebhooksUpdate(data: GatewayPayload) {
   if (data.t !== "WEBHOOKS_UPDATE") return;
 
-  const options = data.d as WebhookUpdatePayload;
+  const options = data.d as WebhookUpdateEventPayload;
```

Comment:
~~`WebhookUpdateEventPayload`~~ `WebhooksUpdateEventPayload`.

Reply:
In docs it is called Webhook Update Event

Reply:
![image](https://user-images.githubusercontent.com/34645569/104050917-cbf15300-5200-11eb-816b-2c0d05af5e9c.png)

Reference: <https://discord.com/developers/docs/topics/gateway#commands-and-events>